modules/nf-core/wipertools/fastqgather/main.nf

```diff
@@ -0,0 +1,48 @@
+process WIPERTOOLS_FASTQGATHER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/wipertools:1.1.3--pyhdfd78af_0':
+        'biocontainers/wipertools:1.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fastq_in)
+
+    output:
+    tuple val(meta), path("${prefix}.fastq.gz") , emit: fastq_out
+    path "versions.yml"                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args    = task.ext.args ?: ''
+    prefix      = task.ext.prefix ?: "${meta.id}"
+    prefix      = prefix + "_gather"
```

Comment:
@SPPearce @tm4zza This is what I meant: you always add `_gather` to the prefix. This needs to be removed I think.